.gitignore

```diff
@@ -21,8 +21,9 @@
 .DS_Store
 *.swp
 
-# binary
-metapod
+#docker-composer
+tmp
```

Comment:
Does docker compose add a tmp file or something?

Reply:
My bad, fresh actually creates to store their binary and error logs. https://github.com/pilu/fresh#usage